swift-paperless/Networking/PaperlessURLSessionDelegate.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  PaperlessURLSessionDelegate.swift
+//  swift-paperless
+//
+//  Created by Nils Witt on 24.06.24.
+//
+
+import Foundation
+import os
+
+class PaperlessURLSessionDelegate: NSObject, URLSessionTaskDelegate {
```

Comment:
```suggestion
final class PaperlessURLSessionDelegate: NSObject, URLSessionTaskDelegate {
```
This needs to be `final`, otherwise it can't conform to `Sendable` (which it must starting in Swift 6)